README.rst

```diff
@@ -118,6 +118,31 @@ Highlights
 
 ----
 
+Who is using DALI:
+------------------
+
+- |kaggle_dali_link|_: |kaggle_great_dali_link|_
+- |neuroscience_DALI|_
+- |HPC_DALI|_
+- |MLPERF_DALI|_
+
+.. |kaggle_dali_link| replace:: It is used during Kaggle computer vision competitions
```

Comment:
```suggestion
.. |kaggle_dali_link| replace:: During Kaggle computer vision competitions
```
Nitpick: It'll look better with all bullets in the same form

Reply:
done